pyEpiabm/pyEpiabm/utility/inverse_cdf.py

```diff
@@ -0,0 +1,63 @@
+#
+# Class of inverse CDF as is done in Covidsim code.
+#
+
+import random
+import pyEpiabm as pe
+import math
+import numpy as np
+
+
+class InverseCdf:
+    """Class of inverse cumulative distribution functions and their associated
+    methods, in a style similar to CovidSim.
+    """
+
+    def __init__(self, mean, icdf_array):
+        """Constructor Method.
+
+        :param mean: Mean of the icdf
+        :type mean: float
+        :param icdf_array: Array of quantiles of the icdf_array
+        :type icdf_array: numpy array
+        """
+        self.mean = mean
+        self.icdf_array = icdf_array
+        self.CDF_RES = pe.Parameters.instance().CDF_RES
+
+    def icdf_choose_noexp(self) -> float:
+        """Chooses a value from the input inverse cumulative distribution function,
+        following what is done in CovidSim (without exponentialisation), and
+        returns the value as a float.
+
+        :returns: chosen value among the quantiles
+        :rtype: float
+        """
+        time_steps_per_day = pe.Parameters.instance().time_steps_per_day
+        rand_num = random.random()
+        q = rand_num*self.CDF_RES
+        i = math.floor(q)
+        q -= float(i)
+        ti = self.mean \
+            * (q * self.icdf_array[i+1] + (1.0 - q) * self.icdf_array[i])
```

Comment:
Try to avoid line breaks with a backslash within mathematical operations (they should only be a last resort anyway, but can look like division here so really hurts readability).

Instead, break around the mathematical operators ([PEP8](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator)).